cves/CVE-2017-7866.yml

```diff
@@ -346,7 +364,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    I think the issue here was just a mistake on the programmer's part in
+    the creation of the stack get. They got the size of the stack off by 1,
+    resulting in this vulnerability. This could have been caused by a lack of
+    understanding of requirements, miscommunication, or just them being forgetful.
```

Comment:
Something about this seems like it's still just a 'coding mistake answer' with some fluff at the end that doesn't mean much.